src/content/reference/hi/p5/orbitControl.mdx

```diff
@@ -176,24 +139,22 @@ class: p5
 params:
   - name: sensitivityX
     description: |
-      <p>sensitivity to movement along the x-axis. Defaults to 1.</p>
+      <p>x-अक्ष के साथ गति के प्रति संवेदनशीलता। डिफ़ॉल्ट 1</p> है
     type: Number
     optional: true
   - name: sensitivityY
     description: |
-      <p>sensitivity to movement along the y-axis. Defaults to 1.</p>
+      <p>y-अक्ष के साथ गति के प्रति संवेदनशीलता। डिफ़ॉल्ट 1</p> है
     type: Number
     optional: true
   - name: sensitivityZ
     description: |
-      <p>sensitivity to movement along the z-axis. Defaults to 1.</p>
+      <p>z-अक्ष के साथ गति के प्रति संवेदनशीलता। डिफ़ॉल्ट 1</p> है
```

Comment:
entire content should be wrapped under <p> tag

Reply:
Done